README.md

```diff
@@ -146,94 +147,107 @@ OpenDataDiscovery Platform offers comprehensive data source support to meet your
-            <td><a href="https://github.com/opendatadiscovery/odd-collector#presto">Presto</a></td>
-            <td><a href="https://github.com/opendatadiscovery/odd-collector#dbt">DBT</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">Trino</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">Presto</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">DBT</a></td>
         </tr>
         <tr>
-            <td><a href="https://github.com/opendatadiscovery/odd-collector#redash">Redash</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">Redash</a></td>
             <td><a href="https://github.com/opendatadiscovery/odd-spark-adapter">Spark</a></td>
-            <td><a href="https://github.com/opendatadiscovery/odd-collector#mlflow">MLflow</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">MLflow</a></td>
         </tr>
         <tr>
-            <td><a href="https://github.com/opendatadiscovery/odd-collector#kubeflow">Kubeflow</a></td>
-            <td><a href="https://github.com/opendatadiscovery/odd-collector#databricks">Databricks Unity Catalog</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">Kubeflow</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">Databricks Unity Catalog</a></td>
             <td><a href="https://github.com/opendatadiscovery/odd-great-expectations">Great Expectations</a></td>
         </tr>
         <tr>
-            <td><a href="https://github.com/opendatadiscovery/odd-collector#sqlite">SQLite</a></td>
-            <td><a href="https://github.com/opendatadiscovery/odd-collector#couchbase">Couchbase</a></td>
-            <td><a href="https://github.com/opendatadiscovery/odd-collector#cockroachdb">Cockroachdb</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">SQLite</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">Couchbase</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">Cockroachdb</a></td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">Fivetran</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">Airbyte</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">Metabase</a></td>
+        </tr>
+        <tr>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector">Mode</a></td>
+            <td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector-azure">BigQuery</a></td>
```

Comment:
Thanks for your fixed for our docs. Found a few links that it would be nice to correct.
BigQuery corresponds to GCP:
`<td><a href="https://github.com/opendatadiscovery/odd-collectors#odd-collector-gcp">BigQuery</a></td>`

Reply:
Thanks, have updated those 👀